config.py

```diff
@@ -90,18 +90,29 @@
 DIVERT_TO_PRIVATE = ('help', )
 
 ROOMS_TO_JOIN = (
```

Comment:
Note that `ROOMS_TO_JOIN` is not an errbot setting.  It is our own variable.  And it is used below and in `plugins/ban.py`

You are removing the prefixes, which means it wont work on Gitter.

So after this, add something like

```py
if BACKEND == 'Gitter':
    ROOMS_TO_JOIN = ['coala/' + item for item in ROOMS_TO_JOIN]
```

Reply:
Added!